pkg/util/pki/common.go

```diff
@@ -49,6 +55,9 @@ const (
 	BrokerControllerFQDNTemplate = "%s.%s.mgt.%s"
 	// CAFQDNTemplate is the template used for the FQDN of a CA
 	CAFQDNTemplate = "%s-ca.%s.cluster.local"
+	// KafkaUserAnnotationName used in case of PKIbackend is k8s-csr to find the appropriate kafkauser in case of
+	// signing request event
+	KafkaUserAnnotationName = "banzaicloud.io/kafkauser-id"
```

Comment:
How about `banzaicloud.io/owner` or `banzaicloud.io/belongs-to`?

This is more generic at allows introducing different type of owners in the future next to KafkaUser